src/pages/Checkout/Checkout.scss

```
@@ -1 +1,136 @@
-//q
+@import '../../styles/mixin';
+@import '../../styles/variables';
+
+.main-checkout {
+  display: flex;
+  justify-content: center;
+  width: 100vw;
+  height: 100vh;
+  padding: 40px;
+  @include textStyle(1em, 500);
+  .article-checkout {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    width: 60%;
+    padding: 40px;
```

```suggestion
.main-checkout {
  @include textStyle(1em, 500);
  display: flex;
  justify-content: center;
  width: 100vw;
  height: 100vh;
  padding: 40px;
  
  .article-checkout {
    display: flex;
    flex-direction: column;
    justify-content: flex-start;
    width: 60%;
    padding: 40px;
```

include는 스타일 블록에서 최상단에 위치시켜 주시고
스타일 블록 끼리는 가독성을 위해 한칸씩 띄워서 작성해주세요!